content/components/augmentedIcon.js

```diff
@@ -0,0 +1,199 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import { bind } from 'decko';
+import { debounce } from 'lodash';
+import { h, Component } from 'preact';
+/** @jsx h */
+
+import AccountCard from './accountCard';
+import Badge from './badge';
+import IdentityIcon from './identityIcon';
+
+import styles from './augmentedIcon.less';
+
+export default class AugmentedIcon extends Component {
+
+  state = {
+    hover: false,
+    open: false
+  };
+
+  constructor (props) {
+    super(props);
+    this.setHover = debounce(this._setHover, 40, { leading: true });
```

Comment:
I think it would be better to call it `setHoverDebounced` or something to distinguish it more from `this._setHover`)